airbyte-webapp/src/components/ui/Toast/Toast.module.scss

```diff
@@ -1,3 +1,89 @@
+@use "scss/colors";
+@use "scss/variables" as vars;
+
+$toast-icon-size: 13px;
+$toast-icon-container-size: 34px;
+
+@keyframes slide-up-animations {
+  0% {
+    transform: translate(-50%, -100%);
+    bottom: -49px;
+  }
+
+  100% {
+    transform: translate(-50%, 0);
+    bottom: 49px;
+  }
+}
+
+@mixin type($name, $color, $background) {
+  &.#{$name} {
+    background-color: $background;
+    border: 1px solid $color;
+
+    .iconContainer {
+      background-color: $color;
+    }
+
+    .toastIcon {
+      color: $color;
+    }
+  }
+}
+
+.toastContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: vars.$spacing-md;
+  position: fixed;
+  box-sizing: border-box;
+  bottom: 49px;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 20;
+  padding: vars.$spacing-md;
+  border-radius: vars.$border-radius-md;
+  animation: slide-up-animations 0.25s linear;
+
+  @include type("info", colors.$blue-400, colors.$blue-50);
+  @include type("warning", colors.$yellow-500, colors.$yellow-50);
+  @include type("success", colors.$green-200, colors.$green-50);
+  @include type("error", colors.$red-300, colors.$red-50);
+}
+
+.iconContainer {
+  width: $toast-icon-container-size;
+  height: $toast-icon-container-size;
+  padding: vars.$border-radius-md;
+  border-radius: vars.$border-radius-md;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.toastIcon {
+  width: $toast-icon-size;
+  height: $toast-icon-size;
+  background: colors.$white;
+  border-radius: 50%;
+}
+
+.text {
+  line-height: 17px;
+  text-align: center;
+}
+
+// style can be removed when Button Dark style will be implemented
+// https://github.com/airbytehq/airbyte/issues/19445
+.actionButton {
+  background-color: colors.$dark-blue-900 !important;
+}
```

Comment:
#19445 has a PR ready to merge, so let's wait for the merge and remove this.

Reply:
Pulled latest changes. Fixed